WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/OrderTotalsCalculator.swift

```diff
@@ -66,7 +68,7 @@ final class OrderTotalsCalculator {
     /// Returns a copy of the order with a new, locally calculated order total.
     ///
     func updateOrderTotal() -> Order {
-        let orderTotal = itemsTotal.adding(shippingTotal).adding(feesTotal).adding(taxesTotal).subtracting(couponsTotal)
+        let orderTotal = itemsTotal.adding(shippingTotal).adding(feesTotal).adding(taxesTotal).subtracting(discountTotal)
```

Comment:
I know this was here before, but the order here doesn't look right to me: Taxes always should be calculated after all price modifications (shipping, fees, coupons, discounts, etc, ...) then over the final net price is where taxes are applied.

Reply:
Thanks! I was overthinking this :D